.travis.yml

```diff
@@ -21,7 +21,7 @@ env:
   # Run all tests before db_block_cache_test (db_test, db_test2)
   - JOB_NAME=unittests ROCKSDBTESTS_END=db_block_cache_test
   # Run all tests starting from db_block_cache_test (db_block_cache_test, ..., plain_table_db_test)
-  - JOB_NAME=unittests ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
+  - JOB_NAME=unittests MEM_ENV=true ROCKSDBTESTS_START=db_block_cache_test ROCKSDBTESTS_END=comparator_db_test
```

Comment:
Makes sense. I would say lets cancel the patch. Only these tests are slow and they need to be tested on travis with real fs.